src/pycram/world_reasoning.py

```diff
@@ -62,6 +63,56 @@ def contact(
             return objects_are_in_contact
 
 
+def robot_will_be_in_collision_at_pose(robot: Object, pose: Pose,
```

Comment:
While the name explains what this method does it is also very long, maybe something like prospection_contact is better.

Reply:
changed to prospect_robot_contact.